sys-libs/libatomic_ops/libatomic_ops-7.4.2.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided \
+             atomic memory update operations on a number architectures \
+             as said on the github project home."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops/"
+COPYRIGHT="1989, 1991 Free Software Foundation, Inc."
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="http://www.ivmaisoft.com/_bin/atomic_ops/libatomic_ops-7.4.2.tar.gz"
+CHECKSUM_SHA256="04fa615f62992547bcbda562260e28b504bc4c06e2f985f267f3ade30304b5dd"
+
+ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+       libatomic_ops$secondaryArchSuffix = $portVersion
+       lib:libatomic_ops$secondaryArchSuffix = $portVersion
+       lib:libatomic_ops_gpl$secondaryArchSuffix = $portVersion
+       "
+REQUIRES="
+       haiku$secondaryArchSuffix
+       "
+
+PROVIDES_devel="
+       libatomic_ops$secondaryArchSuffix_devel = $portVersion
+       devel:libatomic_ops$secondaryArchSuffix = $portVersion
+       devel:libatomic_ops_gpl$secondaryArchSuffix = $portVersion
+       "
+REQUIRES_devel="
+       libatomic_ops$secondaryArchSuffix == $portVersion base
+       "
+
+BUILD_REQUIRES="
+       haiku${secondaryArchSuffix}_devel
+       "
+BUILD_PREREQUIRES="
+       haiku${secondaryArchSuffix}_devel
+       cmd:gcc$secondaryArchSuffix
+       cmd:ld$secondaryArchSuffix
+       cmd:gawk
+       cmd:grep
+       cmd:make
+       cmd:sed
+       cmd:libtool
+       "
+
+BUILD()
+{
+       runConfigure ./configure --enable-shared \
+                                --with-gnu-ld \
+                                —-disable-dependency-tracking
+
+       # Can't run make normally since tests won't build.
```

Comment:
I have just built _x86 version and everything worked. Mark x86_gcc2 as broken (put `!` in front of it).